index.html

```diff
@@ -119,6 +120,13 @@
         cursor: pointer;
         margin: 10px;
       }
+
+      #close {
+        float: right;
```

Comment:
is it possible to use `position: absolute` instead?

Reply:
Done.